src/pages/search.tsx

```diff
@@ -309,6 +313,7 @@ const Search: NextPage<SearchProps> = ({ translations }): JSX.Element => {
                       prefix={<SearchIcon />}
                       onChange={onTranslationSearchQueryChange}
                       onClearClicked={onTranslationSearchClearClicked}
+                      inputRef={searchInputRef}
```

Comment:
Salam alykum Omar,

JAK for your amazing contribution, much appreciated!

The PR looks good masha'Allah. Just a question here. Why are you passing the `searchInputRef` to the translations search input?

Reply:
walaikum assalam Ahmed,

Thanks for your reply.

Sorry about that, i was experiment with it and left it behind i'll remove it.